machines/store.ts

```diff
@@ -153,12 +164,26 @@ export const storeMachine =
                 sendUpdate(),
               ],
             },
+            TAMPERED_VC: {
+              actions: ['setIsTamperedVc'],
+            },
+            RESET_IS_TAMPERED: {
```

Comment:
can we use single action to negate the previous value. 

Instead of 
`setIsTamperedVc: model.assign({
          isTampered: (_, event) => true,
        }),`
        
        
 can we use 
 
 `setIsTamperedVc: model.assign({
          isTampered: (_, event) => !isTampered,
        }),`

Reply:
This has been refactored to use single action for toggling the isTampered.